.github/workflows/ci.yml

```diff
@@ -445,12 +456,14 @@ jobs:
           tags: n0computer/iroh-test:latest
           target: iroh
           platforms: linux/amd64
-          file: docker/Dockerfile
+          file: docker/Dockerfile.ci
 
       - name: Run Docker image & stats test
         run: |
           docker run -p 9090:9090 -p 4919:4919/udp -Pd n0computer/iroh-test:latest --rpc-addr 0.0.0.0:4919 start
-          cargo run --bin iroh -- --rpc-addr 127.0.0.1:4919 stats
+          # Give the server time to start
+          sleep 3
```

Comment:
I predict a future flake :(

Not sure what you can do better though, try to connect to the rpc addr in a loop with a short sleep and overall timeout of 30s?  your connect would have to fail quickly in an identifiable way for that to work though.

Reply:
If this gets flakey we probably should deal with it asap as a 3 sec startup is really not good.
Also the "old" setup just relied on `cargo run` to take it's sweet time compiling.